test/webpack.spec.js

```diff
@@ -1,7 +1,9 @@
 const expect = require('chai').expect;
-const nUiWebpack = require('../build/webpack');
+const nUiWebpack = require('../build/webpack/webpack.config.js');
 
-describe('nUiWebpack', () => {
+// TODO: make this test good, if we can. maybe we can't? ¯\_(ツ)_/¯
```

Comment:
This was brought over from `n-webpack` it possibly won't be as necessary with one large config. I guess it might've been more useful when there was a bit more logic in building the config?